docs/markdown/dune

```diff
@@ -25,13 +25,13 @@
    %{targets}
    (run ocaml-mdx pp %{deps}))))
 
-(rule
- (targets simple_proxy.ml)
- (deps simple-proxy.md)
- (action
-  (with-stdout-to
-   %{targets}
-   (run ocaml-mdx pp %{deps}))))
+;(rule
```

Comment:
What's up with that? The example is not refactored yet?

Reply:
I removed the Morph_client earlier and forgot to remove the example. I should fix this but using Piaf as the client library.

Reply:
But in another PR